ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -39,21 +46,14 @@
 import okhttp3.ws.WebSocket;
 import okhttp3.ws.WebSocketCall;
 import okhttp3.ws.WebSocketListener;
-
-import java.net.URISyntaxException;
-import java.net.URI;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-
 import okio.Buffer;
 import okio.ByteString;
 
 @ReactModule(name = "WebSocketModule")
 public class WebSocketModule extends ReactContextBaseJavaModule {
 
   private final Map<Integer, WebSocket> mWebSocketConnections = new HashMap<>();
+  private final Map<Integer, Boolean> mBlobsEnabled = new HashMap<>();
```

Comment:
Is this a mapping from websocket id to boolean or something? Could you add a comment please? Same with variable above. Probably one comment for both is sufficient.